Cargo.toml

```diff
@@ -5,3 +5,6 @@ members = [
     "android",
     "cpp"
 ]
+
+[profile.release]
+lto = true
```

Comment:
The `-- -C lto` flag no longer works in `build.sh`, and you can only set this at the root `Cargo.toml`. So it will apply to android, cpp, and java when compiled with `--release`. I still think it's worth it as the binary size is reduced by 30%.